site2/docs/functions-api.md

```diff
@@ -265,6 +265,7 @@ public interface Context {
     void incrCounter(String key, long amount);
     long getCounter(String key);
     void putState(String key, ByteBuffer value);
+    void deleteState(String key);
```

Comment:
Since the `functions-api.md` file is deprecated, we can add this in the `functions-develop.md` file.
In the new structure, Functions develop contains information related to available APIs, examples, and other info used for develop Functions.

Reply:
I already added it to functions-develop in bcce54d09

This was just aligning functions-api, but if it's not needed then it should be rm'd.  (It will still exist in repo history).

Reply:
yes, it should be removed.